src/Microsoft.DotNet.Interactive.CSharpProject/Packaging/PackageBase.cs

```diff
@@ -109,6 +114,9 @@ protected async Task DotnetBuildAsync()
                 LastBuildErrorLogFile.Delete();
             }
 
+            // Clean up the temp project file
+            File.Delete(tempDirectoryBuildTarget);

```

Comment:
What is the purpose of the temp file in this method? It looks like we create it and delete it but never use it for anything.

Reply:
This is the Directory.Build.Targets that contains the target to generate the Cache.

We only need this file during the build.

Reply:
Ah, I see. The build reads the file. 👍

Maybe rename it to something that says exactly what it is, e.g. `tempTargetFilePath`

Reply:
done